### PR TITLE
Add manifest.yaml for MongoDB

### PR DESCRIPTION
**Pull request by dmoverton**

### CODE REVIEW

File: plugins/ndc-mongodb/v0.0.1/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+name: mongodb
```
Comment (SandeepSamba) [inline plugins/ndc-mongodb/v0.0.1/manifest.yaml:1]: The file path (which says `ndc-mongodb`) and the name here (which says `mongodb`) must be the same

Comment (SandeepSamba) [reply plugins/ndc-mongodb/v0.0.1/manifest.yaml:1]: Recommend to follow whatever convention `ndc-postgres` team is going with

File: plugins/ndc-mongodb/v0.0.1/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+homepage: https://hasura.io/connectors/mongodb
+hidden: true
+platforms:
+  - selector: darwin-arm64
+    uri: "https://github.com/hasura/ndc-mongodb/releases/download/v0.0.1/mongodb-cli-plugin-aarch64-apple-darwin"
+    sha256: "1c44198b724ed3df406d9fcaa117e0202b19d3fa6ee3ab036a8cdbffbe8aed45"
+    bin: "hasura-mongodb"
+    files:
+      - from: "./mongodb-cli-plugin-aarch64-apple-darwin"
+        to: "hasura-mongodb"
```
Comment (SandeepSamba) [inline plugins/ndc-mongodb/v0.0.1/manifest.yaml:13]: Same, follow same naming conventions uniformly across binary names for all platforms

**Review by SandeepSamba — CHANGES_REQUESTED** [review]
Please follow a unform naming convention. It does not matter if it is `ndc-mongodb` or `mongodb`, but pls ensure it is uniform everywhere

File: plugins/ndc-mongodb/v0.0.1/manifest.yaml (hunk 1 above)
Comment (dmoverton) [reply plugins/ndc-mongodb/v0.0.1/manifest.yaml]: Done

File: plugins/ndc-mongodb/v0.0.1/manifest.yaml (hunk 2 above)
Comment (dmoverton) [reply plugins/ndc-mongodb/v0.0.1/manifest.yaml]: Done

**Pull request merged**